stm32-modules/heater-shaker/simulator/cli_parser.cpp

```diff
@@ -0,0 +1,61 @@
+#include "simulator/cli_parser.hpp"
+#include <boost/program_options.hpp>
+#include <iostream>
+
+using namespace cli_parser;
+
+void no_options_specified_error(boost::program_options::options_description desc) {
+    std::cout << std::endl << "ERROR: You must provide either the --stdin OR the --socket option." << std::endl << std::endl;
```

Comment:
probably wants to be `cerr`

Reply:
This puts it on stderr instead of stdout right?